mypy/nodes.py

```diff
@@ -101,7 +101,8 @@ def get_column(self) -> int:
 implicit_module_attrs = {'__name__': '__builtins__.str',
                          '__doc__': None,  # depends on Python version, see semanal.py
                          '__file__': '__builtins__.str',
-                         '__package__': '__builtins__.str'}
+                         '__package__': '__builtins__.str',
+                         '__path__': '__builtins__.list'}
```

Comment:
IIRC `__path__` is only defined in `__init__.py` modules, not in _every_ module.

Reply:
I thought that `__path__` was defined everywhere. I will fix that.